### PR TITLE
Release MongoDB Kubernetes Operator v0.7.0

### PR DESCRIPTION
**Pull request by killian2k**

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
                 operator: In
                 values:
                 - mongodb-kubernetes-operator
-            topologyKey: "kubernetes.io/hostname"
+            topologyKey: kubernetes.io/hostname
```
Comment (bznein) [inline config/manager/manager.yaml:33]: I assume this change comes from some linting check?

File: docs/RELEASE_NOTES.md [hunk 2]
```diff
@@ -7,12 +7,13 @@
   - Reduce the number of permissions for operator role.
   - Support SHA-1 as an authentication method.
   - Upgraded `mongodbcommunity.mongodbcommunity.mongodb.com` CRD to `v1` from `v1beta1`
-    *  Users upgrading their CRD from v1beta1 to v1 need to set: `spec.preserveUnknownFields` to `false` in the CRD file `config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml` before applying the CRD to the cluster.
+    - Users upgrading their CRD from v1beta1 to v1 need to set: `spec.preserveUnknownFields` to `false` in the CRD file `config/crd/bases/mongodbcommunity.mongodb.com_mongodbcommunity.yaml` before applying the CRD to the cluster.
   - Made service name configurable in mongdb custom resource with statefulSet.spec.serviceName
+  - Add Kube-linter as a github actions.
```
Comment (bznein) [inline docs/RELEASE_NOTES.md:12]: I don't think we ever specified github actions as part of the release notes. I think we focus on more user-facing changes

**Review by bznein — COMMENTED** [review]
LGTM I only have two questions

File: config/manager/manager.yaml (hunk 1 above)
Comment (killian2k) [reply config/manager/manager.yaml:33]: Yes when we ran the update_release.py script

**Review by bznein — APPROVED** [review]

**Pull request merged**